menas/src/main/scala/za/co/absa/enceladus/rest/services/VersionedModelService.scala

```diff
@@ -86,4 +86,11 @@ abstract class VersionedModelService[C <: VersionedModel](versionedMongoReposito
     }
   }
 
+  def validateEntityName(name: String , entityType:String ):String  = {
```

Comment:
```suggestion
  def validateEntityName(name: String, entityType: String): String = {
```
Minor formatting issue.

Reply:
Made the changes.

Reply:
Make `protected`